nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -276,6 +276,9 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - Ada packages (libraries and tools) have been moved into the `gnatPackages` scope. `gnatPackages` uses the default GNAT compiler, `gnat12Packages` and `gnat13Packages` use the respective matching compiler version.
 
+- Paths provided as `restartTriggers` and `reloadTriggers` for systemd units will now be copied into the nix store to make the behavior consistent.
+  Previously, `restartTriggers = [ ./config.txt ]` would either always or never trigger a restart when switching to a new configuration, depending on if the source was in a flake or not.
```

Comment:
Can we settle on this? It's more explicit about the bugged behavior than either previous iteration.

```suggestion
  Previously, `restartTriggers = [ ./config.txt ]`, if defined in a flake, would trigger a restart when any part of the flake changed; and if not defined in a flake, would never trigger a restart even if the contents of `config.txt` changed.
```

Reply:
Yep, looks good.